src/Uno.UI/UI/Xaml/Controls/TreeView/TreeViewNode.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+
+namespace Windows.UI.Xaml.Controls
+{
+	public partial class TreeViewNode : DependencyObject
+	{
+		private TreeViewNode _parent;
+		private bool _hasUnrealizedChildren;
+
+		public TreeViewNode()
+		{
+			var collection = new ObservableVector<TreeViewNode>();
+			//collection.SetParent(this);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)